andela/ah-backend-thor
# #161382391 An Email should be sent to a user upon successful registration. User should be able to receive a verification link via email upon successful signup

#### What does this PR do?
comparing ft-send-email-verify-161382391 with develop
#### Description of Task to be completed?
An Email should be sent to a user upon successful registration. The user should be able to receive a verification link via email upon successful signup
#### How should this be manually tested?
Open Postman register by entering registration details.
check your email and click on the activation link
#### Any background context you want to provide?
TWO STEP AUTHENTICATION
GIVEN I sign up to the Authors Haven Application
WHEN submitted
THEN I should receive a verification email with a link to the application. 
#### What are the relevant pivotal tracker stories?
#161382391
